libs/libuwsc/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libuwsc
-PKG_VERSION:=2.0.4
+PKG_VERSION:=2.0.5
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_VERSION:=v$(PKG_VERSION)
```

Comment:
No, this will clash with other packages.
Use codeload instead
https://github.com/openwrt/packages/blob/master/sound/fdk-aac/Makefile#L15